### ISSUE TITLE
v10.2.1 Override Rates reporting in pence not pounds

### ISSUE THREAD

**Issue by psfshr** [BottlecapDave/HomeAssistant-OctopusEnergy#839]
### Describe the bug

I have recently enabled override rates in the integration, however it appears that rather than reporting in decimal pounds as the normal rates it reports in decimal pence (i.e. 100 times greater).
A manual API call with curl retrieves both in decimal pence.

### Reproduction steps

Retrieving my normal unit rates and agile rates via curl:

```
curl -u "<snip>" "https://api.octopus.energy/v1/products/VAR-22-11-01/electricity-tariffs/E-1R-VAR-22-11-01-H/standard-unit-rates/?period_from=2024-04-15T00:00Z&period_to=2024-04-16T00:00Z" | jq

  "results": [
    {
      "value_exc_vat": 23.482,
      "value_inc_vat": 24.6561,
      "valid_from": "2024-03-31T23:00:00Z",
      "valid_to": null,
      "payment_method": "DIRECT_DEBIT"
    },

curl -u "<snip>" "https://api.octopus.energy/v1/products/AGILE-24-04-03/electricity-tariffs/E-1R-AGILE-24-04-03-H/standard-unit-rates/?period_from=2024-04-15T00:00Z&period_to=2024-04-16T00:00Z" | jq

    {
      "value_exc_vat": 17.22,
      "value_inc_vat": 18.081,
      "valid_from": "2024-04-15T06:30:00Z",
      "valid_to": "2024-04-15T07:00:00Z",
      "payment_method": null
    },

```

Vs the values reported in HA from the integration...

```
previous_day_rates
tariff_code: E-1R-VAR-22-11-01-H
rates:
- start: '2024-04-15T06:30:00+00:00'
  end: '2024-04-15T07:00:00+00:00'
  value_inc_vat: 0.246561
  is_capped: false

previous_consumption_override_rates
tariff_code: E-1R-AGILE-24-04-03-H
rates: 
- value_inc_vat: 18.081
  start: '2024-04-15T06:30:00+00:00'
  end: '2024-04-15T07:00:00+00:00'
  tariff_code: E-1R-AGILE-24-04-03-H
  is_capped: false
```

### Expected behaviour

Integration documentation states that both rates should return decimal pounds values, e.g. would expect:

```
previous_day_rates
tariff_code: E-1R-VAR-22-11-01-H
rates:
- start: '2024-04-15T06:30:00+00:00'
  end: '2024-04-15T07:00:00+00:00'
  value_inc_vat: 0.246561
  is_capped: false

previous_consumption_override_rates
tariff_code: E-1R-AGILE-24-04-03-H
rates: 
- value_inc_vat: 0.18081
  start: '2024-04-15T06:30:00+00:00'
  end: '2024-04-15T07:00:00+00:00'
  tariff_code: E-1R-AGILE-24-04-03-H
  is_capped: false
```

### Tariff Code

As above

### Integration Version

v10.2.1

### Home Assistant Version

2024.4.3

### Fresh Install?

Not specified

### Home Assistant Logs

N/A

### Confirmation

- [X] I confirm that I cannot find my solution within the [documentation](https://bottlecapdave.github.io/HomeAssistant-OctopusEnergy)
- [X] I confirm that I cannot find my solution within the [FAQ](https://bottlecapdave.github.io/HomeAssistant-OctopusEnergy/faq)

**Comment by BottlecapDave** [comment]
Hello and thank you for raising this inconsistency. This should be fixed in the next release, however as it'll include a few breaking changes (including this one), I'm not sure on the timeline at the moment.

**Comment by BottlecapDave** [comment]
This should now be fixed in [v11.0.0](https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/releases/tag/v11.0.0).
